code/modules/mob/living/living.dm

```diff
@@ -7,9 +7,31 @@ Sorry Giacom. Please don't be mad :(
 	if(A && A.push_dir)
 		push_mob_back(src, A.push_dir)
 */
+#define DEFCON1		200
+#define DEFCON2  	400
+#define DEFCON3		600
+#define DEFCON4		800
+#define DEFCON5		1000
+
+/mob/living
+	var/static/mobsInNullSpace = 0
```

Comment:
static is a standard keyword in several languages for a variable shared between all instances of the class. either word is fine; they're synonymous to byond anyway